ext/MitsubaLoader/CMitsubaLoader.h

```diff
@@ -18,6 +18,24 @@ namespace MitsubaLoader
 
 namespace bsdf
 {
+#include "irr/irrpack.h"
+	//must be 64bit
+	struct STextureData
+	{
+		uint16_t pgTab_x;
+		uint16_t pgTab_y;
+		uint16_t width;
+		uint16_t height;
```

Comment:
nice stuff, really like the new API of this commit

one suggestion... make the `width` and `height` store a unorm 16 unsigned ratio of `originalTextureSize/maxAllocatableTextureSize` then we can easily work out the scaling factor for uvs in the virtual texture

`pgTab_x` and y should be unorm offsets in virtual page space, they will be 100% accurate as you require each dimension of virtual page table to be a Power of Two